db/migrate/20210630064201_create_book_authors.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class CreateBookAuthors < ActiveRecord::Migration[6.1]
+  def change
+    create_table :book_authors do |t|
```

Comment:
add unique index for [:book_id, :author_id]

Reply:
done